### ISSUE TITLE
"Please check the value range of input fields" error on remote sensor addition

### ISSUE THREAD

**Issue by bbodenmiller** [tonesto7/nest-manager#356]
## Verify the following before opening an issue:

- [X] That _OAuth_ is Enabled for the SmartApp under the IDE.
- [X] The SmartApp and Device are using the latest version.
- [X] You have reviewed the Readme for potential fixes.
- [X] That the Device Types have been _Published for You_.
## About Your Setup:
- How many Thermostats? 1
- How many Protects? 1
- Are you Using the Nest Presence Device? no
- iOS or Android? Android
- Mobile App Version: 2.2.0
## Expected Behavior

I'm trying to add an automatication to control my thermostat using multiple temperature sensors with the "Remote Temp Sensors" option but when I go to save I'm getting the "Please check the value range of input fields" error message. It appears despite the error that it may be saving.

## Current Behavior

See above.

## Steps to Reproduce (for bugs)

1. Setup Nest Manager
2. Add "Remote Temp Sensors" automation
3. Use values from screenshots
## Context

Please include a copy of your log under the IDE if possible>>>

**Comment by bbodenmiller** [comment]
Can you also clarify the remote sensors automation... once the remote sensor reaches the desired temperature does the thermostat get set back to its original value or how does that work?

**Comment by tonesto7** [comment]
Did you attach screenshots? 
Where you able to capture the error in the IDE? A line number would help me.

**Comment by bbodenmiller** [comment]
![2016-09-28 00 05 27](https://cloud.githubusercontent.com/assets/1192780/18896481/05369c8e-84d7-11e6-9a2e-b1f6f8f5d52e.png)
![2016-09-28 00 05 12](https://cloud.githubusercontent.com/assets/1192780/18896482/0536cb1e-84d7-11e6-8d53-1ce161cb0d81.png)
![2016-09-28 00 05 08](https://cloud.githubusercontent.com/assets/1192780/18896478/05335510-84d7-11e6-93dc-0ebc28962664.png)
![2016-09-28 00 04 56](https://cloud.githubusercontent.com/assets/1192780/18896480/05365e0e-84d7-11e6-8042-6753394859ad.png)
![2016-09-28 00 04 48](https://cloud.githubusercontent.com/assets/1192780/18896479/05345014-84d7-11e6-948a-fb3cc13cdb1c.png)
![2016-09-28 00 04 41](https://cloud.githubusercontent.com/assets/1192780/18896483/054cd36e-84d7-11e6-9f9d-0dd314930953.png)

**Comment by bbodenmiller** [comment]
How can I get the line number? I enabled logging but not seeing it at https://graph.api.smartthings.com/ide/logs. Do I need verbose logging?

**Comment by bbodenmiller** [comment]
Even verbose logging doesn't seem to show line numbers or error messages for this.

**Comment by tonesto7** [comment]
Verbose or debugging should only need to be enabled if you are asked to. 
This sounds like another stupid bug in the Android client related to the input.  
This happens constantly.

Let me replicate your setup on my Android.  If I can find the problem I push out the update.

**Comment by bbodenmiller** [comment]
Bummer. Let me know if I can help at all.

**Comment by tonesto7** [comment]
This is going to take me a few.  I'm current using v4.0 of nest manager which has a completely refactored automation code base. So I can't easily overwrite code without corrupting my current install.   I've asked a colleague if he can try duplicating on his v3.1.1 install.  Once I hear back I will update you.  
Thanks for your patience

**Comment by desertblade** [comment]
I ran through a test. On initial create it worked fine.

On edit, the values zeroed out on temp setting. Once I corrected that it seemed to work well. 

I am on Nest Manager 3.2.  I have seen this a few times on Android, sometimes things don't initialize.  Try deleting the rule, have live logging open and recreated it.  It should have an error somewhere.  If you have it captured ping us and then we can take a look.
